src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String createReport(Map<String, Integer> map) {
```

Comment:
You can go to Storage from this method and you do not need argument
```suggestion
    public String createReport() {
```